client/packages/system/src/Stock/Components/StockLineForm.tsx

```diff
@@ -74,12 +94,6 @@ export const StockLineForm: FC<StockLineFormProps> = ({
     }
   };
 
-  const debouncedUpdate = useDebounceCallback(
-    (patch: Partial<StockLineRowFragment>) => onUpdate(patch),
-    [onUpdate],
-    500
-  );
```

Comment:
Do you think we need to debounce the input? We're only updating the Patch/Draft state, and it seems fine to update this on each keystroke (not really much different to updating a `useState` within a component). What do we think?

Reply:
Seems fine to me 🤷‍♀️ I would have thought we would only need debounce if the onUpdate made an API call, or maybe if it resulted in a big table rerender